netdata/netdata
# feat: move directories to [directories]

##### Summary

This PR:

- Moves out directories and logs-related configuration options to separate sections. It makes it much easier to read the file!
- Backward compatibility included.

<details>
<summary>updated netdata.conf</summary>

```ini
[global]
	# run as user = netdata
	# glibc malloc arena max for plugins = 1
	# glibc malloc arena max for netdata = 1
	# libuv worker threads = 16
	# hostname = deb-work
	# history = 3996
	# update every = 1
	# memory mode = dbengine
	# page cache size = 32
	# dbengine disk space = 256
	# dbengine multihost disk space = 256
	# host access prefix = 
	# memory deduplication (ksm) = yes
	# enable metric correlations = yes
	# TZ environment variable = :/etc/localtime
	# timezone = Europe/Athens
	# OOM score = 0
	# process scheduling policy = batch
	# process nice level = 19
	# pthread stack size = 8388608
	# cleanup obsolete charts after seconds = 3600
	# gap when lost iterations above = 1
	# cleanup orphan hosts after seconds = 3600
	# delete obsolete charts files = yes
	# delete orphan hosts files = yes
	# enable zero metrics = no
	# dbengine extent pages = 64

[directories]
	# config directory = /opt/netdata/etc/netdata
	# stock config directory = /opt/netdata/usr/lib/netdata/conf.d
	# log directory = /opt/netdata/var/log/netdata
	# web files directory = /opt/netdata/usr/share/netdata/web
	# cache directory = /opt/netdata/var/cache/netdata
	# lib directory = /opt/netdata/var/lib/netdata
	# home directory = /var/lib/netdata
	# lock directory = /opt/netdata/var/lib/netdata/lock
	# plugins directory = "/opt/netdata/usr/libexec/netdata/plugins.d" "/opt/netdata/etc/netdata/custom-plugins.d"

[logs]
	# debug flags = 0x0000000000000000
	# debug log = /opt/netdata/var/log/netdata/debug.log
	# error log = /opt/netdata/var/log/netdata/error.log
	# access log = /opt/netdata/var/log/netdata/access.log
	# facility log = daemon
	# errors flood protection period = 1200
	# errors to trigger flood protection = 200
```

</details>

##### Test Plan

- check `/netdata.conf` endpoint.
- check backward compatibility.

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
